packages/protocol/contracts/tokenvault/ERC1155Vault.sol

```diff
@@ -352,13 +350,13 @@ contract ERC1155Vault is BaseNFTVault, ERC1155ReceiverUpgradeable {
                 ctoken.name
             )
         );
-        btoken = address(
-            new TransparentUpgradeableProxy(
-                resolve("proxied_bridged_erc1155", false),
-                owner(),
-                data
-            )
-        );
+
+        btoken = LibDeploy.deployDetermisticUpgradableProxy({
```

Comment:
@adaki2004 please do not merge this PR. 

There maybe an issue with this approach.

Is it possible that someone else deploy a btoken before we do, so the address is already used, so we cannot deploy the contract in vaults?

Reply:
I think it is indeed an issue. We should not merge this PR :)